README.md

```diff
@@ -144,145 +90,52 @@ Example commands:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com: https://github.com:AleMdta/hello-microverse.git
```

Comment:
- [ ]  Kindly narate the Setup of the project explain process how to clone your project repo

Installalation:

Usage:

Run tests: